upgrade_pythoncapi.py

```diff
@@ -563,6 +568,9 @@ def _parse_options(self, args):
         parser.add_argument(
             '-C', '--no-compat', action="store_true",
             help=f"Don't add: {INCLUDE_PYTHONCAPI_COMPAT}")
+        parser.add_argument(
+            '-d', '--download-latest-header', action="store_true",
```

Comment:
Have you considered a shorter long option name, for example `--download` or `--download-header`?